devices/wholeBodyDynamics/GravityCompensationHelpers.cpp

```diff
@@ -0,0 +1,222 @@
+#include "GravityCompensationHelpers.h"
```

Comment:
It must be noted that this class was ported to iDynTree https://github.com/robotology/idyntree/blob/master/src/estimation/include/iDynTree/Estimation/GravityCompensationHelpers.h

Might be useful to refactor out of this repository, in the future.

Reply:
I think we can open issues on this, but I would avoid to mix improvements to the code to just the PR that move the code.

Reply:
Definitely! I agree with that. It's just I noted these details which might be useful if collected together, for the future.